apps/web/src/utils/codeSnippets.ts

```diff
@@ -12,6 +12,13 @@ export type CodeSnippetProps = {
 
 const SECRET_KEY_ENV_KEY = 'NOVU_SECRET_KEY';
 
+const normalizePayload = (originalPayload: Record<string, unknown>) => {
+  // Remove internal params
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const { __source, ...payload } = originalPayload;
+
+  return payload;
+};
```

Comment:
This to not show our internal `__source` in the trigger snippet shown to the user